alias.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller_runtime
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+)
+
+// Builder builds an Application Controller (e.g. Operator) and returns a manager.Manager to start it.
+type Builder = builder.Builder
+
+// Request contains the information necessary to reconcile a Kubernetes object.  This includes the
+// information to uniquely identify the object - its Name and Namespace.  It does NOT contain information about
+// any specific Event or the object contents itself.
+type Request = reconcile.Request
+
+// Result contains the result of a Reconciler invocation.
+type Result = reconcile.Result
+
+type Manager = manager.Manager
+
+type Options = manager.Options
+
+func CreateOrUpdate(ctx context.Context, c client.Client, obj runtime.Object, f controllerutil.MutateFn) (
+	controllerutil.OperationResult, error) {
+	return controllerutil.CreateOrUpdate(ctx, c, obj, f)
+}
+
+func SetControllerReference(owner, object v1.Object, scheme *runtime.Scheme) error {
+	return controllerutil.SetControllerReference(owner, object, scheme)
+}
+
+// Controller returns a new controller builder
+func NewControllerBuilder(m Manager) *Builder {
+	return builder.Controller(m)
+}
+
+// SetupSignalHandler registered for SIGTERM and SIGINT. A stop channel is returned
+// which is closed on one of these signals. If a second signal is caught, the program
+// is terminated with exit code 1.
+func SetupSignalHandler() (stopCh <-chan struct{}) {
+	return signals.SetupSignalHandler()
+}
+
+func GetConfigOrDie() *rest.Config {
+	return config.GetConfigOrDie()
+}
+
+func GetConfig() (*rest.Config, error) {
+	return config.GetConfig()
+}
+
+func NewManager(config *rest.Config, options Options) (manager.Manager, error) {
+	return manager.New(config, options)
+}
```

Comment:
nit: some fns/vars have comments and some don't, so reading the file feels inconsistent. 

Also this will impact the godocs as well.

Reply:
Done.  Removed all comments